### PR TITLE
Feat: Strawpoll form command

### LINKED ISSUE #9: feat: Gather data from strawpoll and print in terminal

### PR DESCRIPTION
**Pull request by Slug-Boi**
Added the CLI command to grab and print result data from a strawpoll. The google cloud auth flow seems extremely hard to work with thus the change to strawpoll if we figure out how to use google instead we can always rework the current methods to use that instead

Resolves #9

### CODE REVIEW

**Comment by Slug-Boi** [comment]
Need tests on this if possible will look into it if not then I will convert to PR

File: src/cmd/form.go [hunk 1]
```diff
@@ -16,16 +17,29 @@ var formCmd = &cobra.Command{
 	You will need to provide the form ID as an argument.
 	The command can be used to retrieve form responses incase you want to pipe it to another command or for testing purposes.
 	`,
-	Args: cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		//TODO: Change to something more meaningful
 		fmt.Println("form called")
```
Comment (Haruyzal) [inline src/cmd/form.go:22]:
```suggestion
		fmt.Println("Form is being processed with the following arguments:", args)
```

**Review by Haruyzal — CHANGES_REQUESTED** [review]

File: src/cmd/form.go (hunk 1 above)
Comment (Slug-Boi) [reply src/cmd/form.go:22]: I agree with the change in message but throwing in the args wouldn't work in cases where the config file is being used instead

Comment (Slug-Boi) [reply src/cmd/form.go]: I have edited the message

**Review by Haruyzal — APPROVED** [review]
Unsure where the Merge conflicts arise though.

**Comment by Slug-Boi** [comment]
> Unsure where the Merge conflicts arise though.

Pretty sure it happened cause i just merged the zap logger PR seems like the go mod and sum files got out of sync cause it adds the zap logger package

**Review by Haruyzal — APPROVED** [review]

**Pull request merged**